_resources/port1.0/group/haskell_cabal-1.0.tcl

```diff
@@ -42,7 +42,10 @@ proc haskell_cabal.add_dependencies {} {
             port:libiconv
     }
     depends_build-append \
-        port:gsed
+        port:cctools \
+        port:file \
+        port:gsed \
+        path:bin/openssl:openssl
```

Comment:
I am not sure this is what we want. Do you mean that an old "openssl" installed by some other means is okay to satisfy the dependency? Why not just depend unconditionally on it?

Reply:
Any `openssl` will do, and I have followed the practice here of other ports that specify the dependency `path:bin/openssl:openssl`.

That said, the intent here is to determine if two files differ within `tcl` (at about line 334 below), and I'm using a RIPEMD hash for that computes by `openssl` exactly as in [macports.tcl](https://github.com/macports/macports-base/blob/49df406d8273f203d30186a77251a897f9d8ef79/src/macports1.0/macports.tcl#L2970).

This is a simple way to accomplish file comparison, but I'm open to a better solution, perhaps just using `diff`.